### ISSUE TITLE
[ERROR] File '...' is missing from the TypeScript compilation.

### ISSUE THREAD

**Issue by kinderswan** [analogjs/analog#460]
### Please provide the environment you discovered this bug in.

`@analogjs/astro-angular": "^0.2.0-beta.16`

### Which area/package is the issue in?

astro-angular

### Description

After including a new version of library to the 'Astro tutorial project' I obtain the following message from console. My tsconfig.app.json is simply copypasted from your base example project,
Also, no angular components exist in the application at the moment, just the config included

`Exlcude: ["node_modules"]` and `skipLibCheck: true` don't change anything

```
import { defineConfig } from 'astro/config';

import angular from "@analogjs/astro-angular";

// https://astro.build/config
export default defineConfig({
  integrations: [angular()]
});
```

### Please provide the exception or error you saw

```true
Sourcemap for "C:/Work/astro-blog/tutorial/node_modules/@analogjs/astro-angular/src/server.js" points to missing source files
X [ERROR] File 'node_modules\rxjs\dist\esm5\index.js' is missing from the TypeScript compilation. [plugin angular-compiler]

  Ensure the file is part of the TypeScript program via the 'files' or 'include' property.

X [ERROR] File 'node_modules\@angular\core\fesm2022\core.mjs' is missing from the TypeScript compilation. [plugin angular-compiler]   

  Ensure the file is part of the TypeScript program via the 'files' or 'include' property.

X [ERROR] File 'node_modules\@angular\platform-browser\fesm2022\platform-browser.mjs' is missing from the TypeScript compilation. [plugin angular-compiler]

  Ensure the file is part of the TypeScript program via the 'files' or 'include' property.

X [ERROR] File 'node_modules\rxjs\dist\esm5\operators\index.js' is missing from the TypeScript compilation. [plugin angular-compiler] 

  Ensure the file is part of the TypeScript program via the 'files' or 'include' property.

X [ERROR] File 'node_modules\@analogjs\astro-angular\src\client.js' is missing from the TypeScript compilation. [plugin angular-compiler]

  Ensure the file is part of the TypeScript program via the 'files' or 'include' property.

node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error: Build failed with 5 errors:
error: File 'node_modules\rxjs\dist\esm5\index.js' is missing from the TypeScript compilation.
error: File 'node_modules\@angular\core\fesm2022\core.mjs' is missing from the TypeScript compilation.
error: File 'node_modules\@angular\platform-browser\fesm2022\platform-browser.mjs' is missing from the TypeScript compilation.        
error: File 'node_modules\rxjs\dist\esm5\operators\index.js' is missing from the TypeScript compilation.
error: File 'node_modules\@analogjs\astro-angular\src\client.js' is missing from the TypeScript compilation.
    at failureErrorWithLog (C:\Work\astro-blog\tutorial\node_modules\esbuild\lib\main.js:1636:15)
    at C:\Work\astro-blog\tutorial\node_modules\esbuild\lib\main.js:1048:25
    at C:\Work\astro-blog\tutorial\node_modules\esbuild\lib\main.js:1512:9
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  errors: [
    {
      detail: undefined,
      id: '',
      location: null,
      notes: [
        {
          location: null,
          text: "Ensure the file is part of the TypeScript program via the 'files' or 'include' property."
        }
      ],
      pluginName: 'angular-compiler',
      text: "File 'node_modules\\rxjs\\dist\\esm5\\index.js' is missing from the TypeScript compilation."
    },
    {
      detail: undefined,
      id: '',
      location: null,
      notes: [
        {
          location: null,
          text: "Ensure the file is part of the TypeScript program via the 'files' or 'include' property."
        }
      ],
      pluginName: 'angular-compiler',
      text: "File 'node_modules\\@angular\\core\\fesm2022\\core.mjs' is missing from the TypeScript compilation."
    },
    {
      detail: undefined,
      id: '',
      location: null,
      notes: [
        {
          location: null,
          text: "Ensure the file is part of the TypeScript program via the 'files' or 'include' property."
        }
      ],
      pluginName: 'angular-compiler',
      text: "File 'node_modules\\@angular\\platform-browser\\fesm2022\\platform-browser.mjs' is missing from the TypeScript compilation."
    },
    {
      detail: undefined,
      id: '',
      location: null,
      notes: [
        {
          location: null,
          text: "Ensure the file is part of the TypeScript program via the 'files' or 'include' property."
        }
      ],
      pluginName: 'angular-compiler',
      text: "File 'node_modules\\rxjs\\dist\\esm5\\operators\\index.js' is missing from the TypeScript compilation."
    },
    {
      detail: undefined,
      id: '',
      location: null,
      notes: [
        {
          location: null,
          text: "Ensure the file is part of the TypeScript program via the 'files' or 'include' property."
        }
      ],
      pluginName: 'angular-compiler',
      text: "File 'node_modules\\@analogjs\\astro-angular\\src\\client.js' is missing from the TypeScript compilation."
    }
  ],
  warnings: [
    {
      detail: undefined,
      id: '',
      location: {
        column: 0,
        file: 'tsconfig.app.json',
        length: 0,
        line: 0,
        lineText: '',
        namespace: 'file',
        suggestion: ''
      },
      notes: [
        {
          location: null,
          text: 'To control ECMA version and features use the Browerslist configuration. For more information, see https://angular.io/guide/build#configuring-browser-compatibility'
        }
      ],
      pluginName: 'angular-compiler',
      text: 'TypeScript compiler options "target" and "useDefineForClassFields" are set to "ES2022" and "false" respectively by the Angular CLI.'
    }
  ]
}
```

### Other information

Error: https://pastebin.com/EFKuhyKy

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [ ] No

**Comment by Drunkenpilot** [comment]
Got same issue after update to 0.2.0-beta.16

**Comment by Drunkenpilot** [comment]
The issue only showed up when Astro version above 2.6.0
Astro@2.5.7 has no issue.

**Comment by brandonroberts** [comment]
This should be resolved in the latest `0.2.0-beta.17` release with/without the `allowJs` flag
